app/routes/circulars.$circularId.($version)/route.tsx

```diff
@@ -154,10 +156,13 @@ function CircularsHistory({
   circular: number
   history?: number[]
 }) {
-  const [showVersions, setShowVersions] = useState<boolean>(false)
-
+  const ref = useRef<HTMLDivElement>(null)
+  const [showVersions, setShowVersions] = useState(false)
```

Comment:
```suggestion
  const [showContent, setShowContent] = useState(false)
```

Reply:
It's your PR. Please just go ahead and make those changes.